### ISSUE TITLE
For the people who suddenly found the hot reload on save not work on Android Studio

### ISSUE THREAD

**Issue by Cookiezby** [flutter/flutter-intellij#7063]
Thanks for the feedback! If your issue is related to the Flutter framework itself,
please open an issue at
[github.com/flutter/flutter](https://github.com/flutter/flutter/issues/new).

## Steps to Reproduce

It spend me several hours to figure out the reason, so I hope this would help to save your time

Downloading the 76.0.x or earlier zip file from this link
https://plugins.jetbrains.com/plugin/9212-flutter/versions/stable

And install the plugin manually from file, which will make the hot reload works again

<img width="967" alt="Screenshot 2023-11-06 at 0 54 03" src="https://github.com/flutter/flutter-intellij/assets/5690511/abe14c90-bdb1-4eca-912c-138fa01ad675">

## Version info

Please paste the output of running `flutter doctor -v` here (available from the command
line or from `Tools > Flutter > Flutter Doctor`). It will provide the version of the
Flutter framework as well as of the IntelliJ plugin.

**Comment by billliyu** [comment]
I tried rolling back the version to 2023.1 on Intellij Idea, the flutter plugin changed to 76.1.3, and the hot reload feature did not work.
However my Android Sutdio IDE works, although the version is Android Studio Giraffe | 2022.3.1 Patch 2 (223.8836.) and the flutter plugin version is 75.1.2 and it works fine.
Using version 76.0.x would mean that IDEA would have to fall back to version 2022, which would cause the other plugins to be downgraded along with it.

**Comment by safiul-incrementsinc** [comment]
facing same issue here 
```[✓] Flutter (Channel stable, 3.13.2, on macOS 14.0 23A344 darwin-arm64, locale en-BD)
    • Flutter version 3.13.2 on channel stable at /Users/mac4ipp/Documents/dev/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ff5b5b5fa6 (2 months ago), 2023-08-24 08:12:28 -0500
    • Engine revision b20183e040
    • Dart version 3.1.0
    • DevTools version 2.25.0

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/mac4ipp/Library/Android/sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.0)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15A240d
    • CocoaPods version 1.14.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2022.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)

[✓] VS Code (version 1.84.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.76.0

[✓] VS Code (version 1.81.1)
    • VS Code at /Users/mac4ipp/Downloads/Visual Studio Code.app/Contents
    • Flutter extension version 3.76.0

[✓] Connected device (4 available)
    • sdk gphone64 arm64 (mobile) • emulator-5554                        • android-arm64  • Android 14 (API 34) (emulator)
    • iPhone 15 Pro (mobile)      • 018A85B5-6EFA-4AE1-92A3-9E1822E5595B • ios            • com.apple.CoreSimulator.SimRuntime.iOS-17-0 (simulator)
    • macOS (desktop)             • macos                                • darwin-arm64   • macOS 14.0 23A344 darwin-arm64
    • Chrome (web)                • chrome                               • web-javascript • Google Chrome 119.0.6045.105

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```

**Comment by neldeles** [comment]
> I tried rolling back the version to 2023.1 on Intellij Idea, the flutter plugin changed to 76.1.3, and the hot reload feature did not work. However my Android Sutdio IDE works, although the version is Android Studio Giraffe | 2022.3.1 Patch 2 (223.8836.) and the flutter plugin version is 75.1.2 and it works fine. Using version 76.0.x would mean that IDEA would have to fall back to version 2022, which would cause the other plugins to be downgraded along with it.


There's a version that works with the latest IDEA. 
![clop_2023-11-08_94](https://github.com/flutter/flutter-intellij/assets/14298160/4020a303-ca2c-4a7c-b828-3eb83c62567c)
I am on 2023.2.2 and downgraded to Flutter plugin version 76.0.4. Hot reloading on save is fine with this version.

**Comment by billliyu** [comment]
> There's a version that works with the latest IDEA.

Thanks, hot reload feature works well in IDEA 2023.2.4, Flutter plugin 76.0.4

**Comment by jwren** [comment]
Hi all, I believe that this issue is fixed with the point release of `76.3.x` that I just pushed out. Please let me know otherwise.

Thank you for the various reports and comments above.
